proxy/proxy.go

```diff
@@ -252,11 +252,19 @@ func newProxy() *proxy {
 
 // This variable is populated at link time with the value of:
 //   ${locatestatedir}/run/cc-oci-runtime/proxy
-var socketPath string
+var DefaultSocketPath string
```

Comment:
```
cc-oci-runtime-0.1/proxy/proxy.go:253:1: comment on exported var DefaultSocketPath should be of the form "DefaultSocketPath ..."
cc-oci-runtime-0.1/proxy/proxy.go:257:1: comment on exported var ArgSocketPath should be of the form "ArgSocketPath ..."
```

You could either change the comments to start like go comments golint expects, or (better IMO), don't expose those variables outside of the package.